packages/liveblocks-core/src/types/IWebSocket.ts

```diff
@@ -75,6 +75,8 @@ export enum WebsocketCloseCodes {
   MAX_NUMBER_OF_MESSAGES_PER_DAY_PER_APP = 4004,
   /** Room is full, disconnect */
   MAX_NUMBER_OF_CONCURRENT_CONNECTIONS_PER_ROOM = 4005,
+  /** The room's ID was updated, disconnect */
+  ROOM_ID_UPDATED = 4006,
```

Comment:
Nice! Picking a code in the 40xx range will mean older clients are going to disconnect permanently and not try to reconnect to the new room ID. And if you use a newer client, it can reconnect to the new room ID automatically. Elegant solution! 💯